nimx/render_to_image.nim

```diff
@@ -130,6 +131,8 @@ proc beginDraw*(t: ImageRenderTarget, state: var GlFrameState) =
 
     let gl = sharedGL()
     state.framebuffer = gl.boundFramebuffer()
+    when defined(ios):
+        state.renderBuffer = cast[RenderbufferRef](gl.getParami(GL_FRAMEBUFFER_BINDING))
```

Comment:
Whats the sense of this? You're getting framebuffer, pretending that is renderbuffer, and leter bind it back as `RENDERBUFFER`. Btw, `cast[RenderbufferRef](gl.getParami(GL_FRAMEBUFFER_BINDING))` is 100% equivalent to `gl.boundFramebuffer()`